streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/desc/ListStateDescriptor.java

```diff
@@ -0,0 +1,66 @@
+package org.ray.streaming.state.keystate.desc;
+
+
+import org.ray.streaming.state.keystate.state.ListState;
+
+/**
+ * ListStateDescriptor.
+ */
+public class ListStateDescriptor<T> extends AbstractStateDescriptor<ListState<T>, T> {
+
+  private boolean isOperatorList;
```

Comment:
I think these fields are `final` and thus nobody should use the setters.

Reply:
ok, i'll fix this